src/Sylius/Bundle/ResourceBundle/DependencyInjection/Driver/AbstractDriver.php

```diff
@@ -175,11 +183,37 @@ protected function addForms(ContainerBuilder $container, MetadataInterface $meta
                 sprintf('%s.form.type.%s%s', $metadata->getApplicationName(), $metadata->getName(), $suffix),
                 $definition
             );
+        }
 
+        if (!$container->hasDefinition(sprintf('%s.form.type.%s', $metadata->getApplicationName(), $metadata->getName()))) {
```

Comment:
ok, you are overwriting my already registered form type service here, i'm trying to track down why this `hasDefinition` check returns false.